haystack/preview/README.md

```diff
@@ -1 +1,27 @@
-# Haystack - Preview features
+# Haystack 2.0 - Preview Features
+
+[![PyPI - Version](https://img.shields.io/pypi/v/haystack-ai.svg)](https://pypi.org/project/haystack-ai)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/haystack-ai.svg)](https://pypi.org/project/haystack-ai)
+
+Since Haystack v1.15, we’ve been slowly introducing new components and features to Haystack in the background in preparation for Haystack 2.0. In this `preview` module, you can find what’s been implemented so far regarding Haystack 2.0. **Keep in mind that Haystack 2.0 is still a work in progress.** Read more about Haystack 2.0 in [Shaping Haystack 2.0](https://github.com/deepset-ai/haystack/discussions/5568).
+
+## 💾 Installation
+
+**Install `haystack-ai`**
+
+There is a separate PyPI package that only ships the code in `preview` module. You can install `haystack-ai` using pip:
+```sh
+pip install haystack-ai
+```
+The `haystack-ai` package is built on the `main` branch, so it's highly unstable, but it's useful if you want to try the new features as soon as they are merged. 
+
+**Install `farm-haystack`**
+
+As an alternative way, you can install `farm-haystack`:
+```sh
+pip install farm-haystack
+```
+The `farm-haystack` package includes all new features regarding Haystack 2.0. Note that updates to this package occur less frequently compared to `haystack-ai`. So, you might not get the latest Haystack 2.0 features when using `farm-haystack`.
```

Comment:
```suggestion
The `farm-haystack` package includes all new features of Haystack 2.0. Note that updates to this package occur less frequently compared to `haystack-ai`. So, you might not get the all latest Haystack 2.0 features immediately when using `farm-haystack`.
```

I think that the original text contradicts a little bit, saying that `farm-haystack` includes all new features, but you might not get them :D So I tried rephrasing this a bit with this suggestion, I hope it's accurate?

Reply:
@dfokina Yeap, it sounds much better!